tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py34,py35,py36,pep8
+
+[testenv]
+extras =
+    test
+deps =
+    coverage
+commands =
+    pip list
+    coverage run -m pytest {posargs}
+    coverage report -m
```

Comment:
We can, I'm just still biased towards using `coverage` directly 😛 but don't have a strong argument.